tests/ast/data/DataFrame.to_pandas.test

```diff
@@ -0,0 +1,39 @@
+## TEST CASE
+
+df = session.table("test_table")
+
+pandas_df = df.to_pandas()
+
+pandas_df = df.to_pandas(block=False)
+
+pandas_df = df.to_pandas(statement_params={"SF_PARTNER": "FAKE_PARTNER"})
+
+pandas_df = df.to_pandas(statement_params={"SF_PARTNER": "FAKE_PARTNER"}, block=False)
+
+## EXPECTED UNPARSER OUTPUT
+
+df = session.table("test_table")
+
+pandas_df = Add a handler for Empty$ in Unparser.spDataframeExpr().to_pandas()
+
+pandas_df
+
```

Comment:
missing handler? need to update expectations? Can you sync with your snowpark change/rebuild the unparser?